virttest/unittests/libvirt_xml/test_vm_xml.py

```diff
@@ -0,0 +1,135 @@
+import unittest
+
+from virttest.libvirt_xml import vm_xml, xcepts
+
+XML = """
```

Comment:
How about put those config into separate xml file ? you can refer to https://github.com/autotest/tp-libvirt/pull/2216, for how access this file.

Reply:
@chunfuwen I'd like to not load the file from disk as it's currently not needed in other cases. Instead, I reduced the xml snippet to only contain the xml relevant for the test. What do you think?

```bash
# python -W ignore -m unittest test_vm_xml.py -v
test_del_seclabel_default (test_vm_xml.TestVMXMLDelSeclabel) ... ok
test_del_seclabel_with_conditions (test_vm_xml.TestVMXMLDelSeclabel) ... ok
test_del_seclabel_with_partial_match (test_vm_xml.TestVMXMLDelSeclabel) ... ok

----------------------------------------------------------------------
Ran 3 tests in 0.083s

OK
```